fetch-uj-records/fetch-uj-records_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/redhat-appstudio/segment-bridge.git/containerfixture"
+	"github.com/redhat-appstudio/segment-bridge.git/splunk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	scriptPath   = "../scripts/fetch-uj-records.sh"
+	filePathPass = "./fetch-uj-recordsPass"
+)
+
+func compareOutputs(output []byte, filePath string) bool {
+	/*Recieves the script's output and compares it
+	with a file containing the expected output*/
+	fileOutput, err := os.ReadFile(filePath)
+	if err != nil {
+		fmt.Printf("error: %s\n", err.Error())
```

Comment:
Use `t *testing.T` and t.Log instead of Printf
Apply this to all other Printf in the test files.